packages/fxa-payments-server/server/config/index.js

```diff
@@ -272,7 +272,7 @@ const conf = convict({
   },
   stripe: {
     apiKey: {
-      default: 'pk_test_FL2cOisOukoCQUZsrochvTlk00ff4IakfE',
+      default: 'pk_test_UrTy6gLWGrc0aOHtbZZz0UBs005mSI4AS2',
```

Comment:
IIRC, the change here is to stop using my personal Stripe key (😬) and switch to what our dev deployment uses.